pkg/controller/reconciler/deployment_reconciler_test.go

```diff
@@ -446,6 +446,44 @@ func Test_DeploymentReconciler_Connections(t *testing.T) {
 	require.True(t, apierrors.IsNotFound(err))
 }
 
+// Creates a deployment with Radius disabled.
+//
+// Then checks the Events for Disabled-Disabled.
```

Comment:
I'd like to see the same test for the case where no annotations are present.

Reply:
Added